src/components/Input/index.tsx

```diff
@@ -0,0 +1,42 @@
+interface InputProps {
+  disabled: boolean;
+  label?: string;
+  essential: boolean;
+  limit?: number | undefined;
+  baseData?: string;
+  width: 'small' | 'medium' | 'large';
+}
+
+const Input = ({
+  disabled,
+  label,
+  essential,
+  limit,
+  baseData,
+  width,
+}: InputProps) => {
+  const widthType = {
+    small: 'w-[100px]',
+    medium: 'w-[240px]',
+    large: 'w-[358px]',
+  };
+
+  return (
+    <div className={`flex ${widthType[width]} flex-col`}>
+      <div className='mb-2 flex justify-between'>
+        <label className='text-gray-accent2'>
+          {label}
+          {essential && <span className='ml-0.5 text-red-500'>*</span>}
+        </label>
+        {limit && <label className='text-gray-accent2'>{limit}</label>}
+      </div>
+      <input
+        placeholder={baseData}
+        disabled={disabled}
+        className={`h-[55px] ${widthType[width]} rounded-lg border-2 border-gray-accent7 text-gray-accent2 ${disabled ? 'bg-gray-accent7' : ''} pl-5`}
```

Comment:
focus 됐을 때 outline 색상을 primary로 바꾸면 좋을 것 같아요!
```suggestion
        className={`h-[55px] ${widthType[width]} rounded-lg border-2 border-gray-accent7 text-gray-accent2 ${disabled ? 'bg-gray-accent7' : ''} pl-5 focus:outline-primary`}
```
<img src="https://github.com/BoardSignal/Team-CivilWar-BoardSignal-FE/assets/42732729/740f3a74-b93c-4322-87ef-9f6a80e4153f" width="20%">

Reply:
저는 희라님이 말씀해주신 부분 이렇게 적용했습니다! 만약 적용안되시면 이 코드도 적용해보세요 
`outline-none focus:border-violet-500`

Reply:
Input 상태까지는 와이어프레임을 안 만들어놓아서 결정이 안 된 부분인데 저는 `focus:outline-gray-accent2`도 좋을 것 같아요 :)